README.md

```diff
@@ -29,7 +29,7 @@ yarn add puppeteer
 # or "npm i puppeteer"
 ```
 
-> **Note**: When you install Puppeteer, it downloads a recent version of Chromium (~71Mb Mac, ~90Mb Linux, ~110Mb Win) that is guaranteed to work with the API.
+> **Note**: When you install Puppeteer, it downloads a recent version of Chromium (~71Mb Mac, ~90Mb Linux, ~110Mb Win) that is guaranteed to work with the API. Although it's not recommend, you can skip the Chromium install with `npm i --ignore-scripts puppeteer`.
```

Comment:
recommended

Reply:
is this enough? If I want to deploy an app that depends on Puppeteer in its `package.json`, than I can't use the `npm i --ignore-scripts puppeteer`